packages/spruce-skill-server/interfaces/ctx.ts

```diff
@@ -16,16 +16,23 @@ export interface ISpruceContext<
 	ISkillUtilities = ISpruceUtilities,
 	ISkillGQLTypes = ISpruceGQLTypes
 > extends Context, Router.RouterContext {
+	// run operations against core
```

Comment:
If you want these to show up in code completion I think you need to use the jsdoc format:
`/** Comment here */`